### PR TITLE
Add signup

### PR DESCRIPTION
**Pull request by tlim045**
- Added checkboxes to differentiate researchers/volunteers.
- No routing implemented to access sign up page yet.
- Next PR will include extended sign up form for volunteers to input personal details.

### CODE REVIEW

File: src/onepirate/modules/form/CheckBox.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import Grid from '@material-ui/core/Grid';
+
+export default function CheckboxLabels() {
```
Comment (haveaqiupill) [inline src/onepirate/modules/form/CheckBox.js:7]: Would radio buttons be better as they are mutually exclusive 😮 I can select both "Researcher" and "Volunteer" in the form but I think they should be mutually exclusive: 
image

**Comment by haveaqiupill** [comment]
The rest LGTM!

File: src/onepirate/modules/form/CheckBox.js (hunk 1 above)
Comment (tlim045) [reply src/onepirate/modules/form/CheckBox.js:5]: I think radio buttons are better too, thanks for the suggestion! Made the change already.

**Comment by haveaqiupill** [comment]
@wennyyustalim not yet! I'll add it in

**Pull request merged**